### DISCUSSION TITLE
Section 4.7 of Modeling Information with the Common Core Ontologies 1.0

### DISCUSSION THREAD

**Discussion by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#363] [Announcements]
In the diagram it is the information bearer that is associated with the value and measurement unit. I would think that it is the information content entity that those are associated with. Would not all copies (concretizations) of the information content entity on different bearers have the same value and measurement unit?

**Comment by oliviahobai** [comment]
Discussion on the domain of value/measurement is to continue here
